src/ontology/cl-edit.owl

```diff
@@ -28486,6 +28487,16 @@ SubClassOf(obo:CL_4023181 obo:CL_0000710)
 SubClassOf(obo:CL_4023181 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002139))
 SubClassOf(obo:CL_4023181 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015658))
 
+# Class: obo:CL_4023187 (koniocellular cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Lateral_geniculate_nucleus&oldid=1098375993"^^xsd:string) obo:IAO_0000115 obo:CL_4023187 "A neuron with a small cell body that is located in a koniocellular layer of the lateral geniculate nucleus (LGN).")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023187 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023187 "NIFEXT:44")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10845061"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4023187 "K cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:10845061"^^xsd:string) rdfs:comment obo:CL_4023187 "In adult monkeys, koniocellular cells can be distinguished from parvocellular and magnocellular of the LGN by their expression of calbindin, the alpha subunit of type II calmodulin-dependent protein kinase, and the gamma subunit of protein kinase C.")
+AnnotationAssertion(rdfs:label obo:CL_4023187 "koniocellular cell")
+EquivalentClasses(obo:CL_4023187 ObjectIntersectionOf(obo:CL_0000540 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0013615)))
```

Comment:
koniocellular layer of dorsal nucleus of lateral geniculate body'

Are all koniocelluar layers in the dorsal nucleus of the LGB?

Reply:
> Are all koniocelluar layers in the dorsal nucleus of the LGB?

yes lol, this took me awhile to get to, but after looking at literature + allen human brain atlas, it is

Reply:
just to be extra sure, I won't merge till @patrick-lloyd-ray gives a thumbs up :)

Reply:
LGTM.